### PR TITLE
[NEXUS-8655] LDAP's Verify Connection times out faster than listed

### PR DESCRIPTION
**Pull request by adreghiciu**
https://issues.sonatype.org/browse/NEXUS-8655

### CODE REVIEW

File: plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/nexus/ldap/internal/realms/LdapConnectionUtils.java [hunk 1]
```diff
@@ -33,28 +39,33 @@
 
 public class LdapConnectionUtils
 {
-  private static Logger logger = LoggerFactory.getLogger(LdapConnectionUtils.class);
+  private static Logger log = LoggerFactory.getLogger(LdapConnectionUtils.class);
 
-  public static DefaultLdapContextFactory getLdapContextFactory(LdapConfiguration ldapServer)
+  public static LdapContextFactory getLdapContextFactory(final LdapConfiguration configuration,
```
Comment (mrprescott) [inline plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/nexus/ldap/internal/realms/LdapConnectionUtils.java]: Suggest @Nullable for 'configuration' parameter, and checkNotNull(trustStore)

**Comment by mrprescott** [comment]
+1

**Comment by cstamas** [comment]
+1

**Comment by mcculls** [comment]
+1

**Comment by adreghiciu** [comment]
Manually merged as https://github.com/sonatype/nexus-oss/commit/8a77681eb336d25fc0caae71033ed570210961b5

**Pull request closed without merging**